ra-optimisation/linear-range-action-rao/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, RTE (http://www.rte-france.com)
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~  License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>farao-ra-optimisation</artifactId>
+        <groupId>com.farao-community.farao</groupId>
+        <version>1.3.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>farao-linear-range-action-rao</artifactId>
+    <packaging>jar</packaging>
+    <name>Modular RAO on linear actions implementation</name>
```

Comment:
I would say that the optimisation is linear, but not the actions. We approximate their impact with linear sensitivity coefficients, yet they are _a priori_ not linear.
Maybe : 
`Modular linear RAO implementation`